bin/MadGraph5_aMCatNLO/cards/production/13TeV/DYJets_LO_MLM/DY_HT-100to200_proc_card.dat

```diff
@@ -0,0 +1,43 @@
+#************************************************************
+#*                        MadGraph 5                        *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 1.5.11                2013-06-21         *
+#*                                                          *
+#*    The MadGraph Development Team - Please visit us at    *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph 5                *
+#*                                                          *
+#*     run as ./bin/mg5  filename                           *
+#*                                                          *
+#************************************************************
+
+import model sm-no_masses
```

Comment:
Default for 5-flavour should be sm-ckm_no_b_mass

clearly the ckm part makes no difference for drell-yan, but the implications of passing massless tau's to pythia have not been checked, so prefer not to do it.